typescript/web/src/components/labeling-tool/openlayers-map/openlayers-map.tsx

```diff
@@ -322,7 +322,7 @@ export const OpenlayersMap = () => {
         !isContextMenuOpen && <CursorGuides map={mapRef.current} />}
       {/* This div is needed to prevent a weird error that seems related to the EditLabelClass component */}
       <Box
-        key="toto"
+        key="FIXME"
```

Comment:
I think we can completely remove that key.

By the way, the comment in the previous line refers to a div that is not in the code anymore, or at least it has been converted in a `Box` which now has a real purpose. It comes from [that commit](https://github.com/labelflow/labelflow/commit/8b4e5144b3674a37c38e85e1cbadf44f5074cc7c)

Reply:
I've updated the code following your remarks about this file.